incident/api/tests/test_csv.py

```diff
@@ -288,14 +287,6 @@ def test_arresting_authority_is_correct(self):
             'Police Squad!',
         )
 
-    def test_status_of_charges_is_correct(self):
-        self.assertEqual(
-            self.result['status_of_charges'],
-            ', '.join(
-                charge.updates.order_by('-date').first().get_status_display() for charge in self.incident.charges.all()
-            )
-        )
-
```

Comment:
I think we do want to preserve this test, as it does not depend on the `status_of_charges` field directly, and instead uses the new style charge metadata.